app/src/modules/settings/components/navigation.vue

```diff
@@ -27,9 +27,14 @@ const links = computed<Link[][]>(() => [
 		},
 		{
 			icon: 'admin_panel_settings',
-			name: t('settings_permissions'),
+			name: t('settings_roles'),
 			to: `/settings/roles`,
 		},
+		{
+			icon: 'badge',
```

Comment:
Lets use `admin_panel_settings` as the icon here

```suggestion
			icon: 'admin_panel_settings',
```